src/components/navigation/nav-tab-panel.tsx

```diff
@@ -55,9 +56,11 @@ export class NavTabPanel extends BaseComponent<IProps, IState> {
                               ? `calc(100% - ${collapseTabWidth}px - ${resizePanelWidth}px)`
                               : `calc(${dividerPosition}% - ${resizePanelWidth}px)`;
     const resourceWidthStyle = {width: resourceWidth};
+    const showChatPanel = this.state.showChatColumn;
+    const isChatEnabled = user.isNetworkedTeacher && urlParams.chat;
```

Comment:
I would have thought that what we wanted was:
```suggestion
    const isChatEnabled = user.isNetworkedTeacher && urlParams.chat;
    const showChatPanel = isChatEnabled && this.state.showChatColumn;
```
Otherwise, there's at least a theoretical possibility that `showChatPanel` will be true when `isChatEnabled` is false and then things get shown even though the feature is disabled.